simplepso/pso.py

```diff
@@ -25,8 +36,9 @@ class PSO(object):
         To run need to supply number of particles and number of iterations. 20 particles is a good starting place.
 
     """
+
     def __init__(self, cost_function=None, start=None, num_proc=1,
-                 save_sampled=False, verbose=False):
+                 save_sampled=False, verbose=False, shrink_steps=True):
```

Comment:
I see num_proc is not being used anymore. Maybe you should delete this parameter from __init__ and the documentation. Also, you should probably document the shrink_steps parameter